release-tools/scripts/prod-sync-artifacts.py

```diff
@@ -0,0 +1,265 @@
+        'Key': plugin_val}
+        print("Upload plugin : " + plugin_val)
+        artifact_name = plugin_val.split('/')[-1]
+        print("Artifact initial name : " + artifact_name) 
+        if "-build-" in plugin_val: 
+            artifact_name = re.sub('-build-[0-9]*','',artifact_name)
+        print("##### Upload in progress #####")
+        print("Plugin name : " + plugin_val)
+        print("Artifact name : " + artifact_name)
+        print("Source bucket :" + rc_bucket)
+        print("Destination path : " + folder_path)
+        print("Destination bucket name " + bucket_name)
+        response = s3.copy_object(CopySource=copy_source, Bucket=bucket_name, 
+                Key=folder_path + artifact_name, MetadataDirective="REPLACE")
+        if response.get('ResponseMetadata').get('HTTPStatusCode') == 200:
+            upload_list.append([artifact_name,"Upload Successful"])
+        else:
+            upload_list.append([artifact_name,"Upload Failed"])
+    except:
+        raise
+
+# Check if the ODFE artifact is available
+def check_odfe(rc_bucket,odfe_path,artifact_type,odfe_name):
+    try:
+        response = s3.list_objects_v2(Bucket=rc_bucket, Prefix=odfe_path)
+        plugin = []
+        for artifact in sorted(response['Contents'], key=lambda x: x['LastModified'], reverse=True):
+            key = artifact['Key']
+            if (odfe_name in key):
+                if (key.endswith(artifact_type) or key.endswith(artifact_type + ".sha512")):
+                    plugin.append(key)
+        return plugin
+    except:
+        raise
+
+
+# Check if the Plugin is available
+def check_plugin(rc_bucket,plugin_fullpath,plugin_name,plugin_ver,plugin_build,key_word):
+    try:
+        bld_num = "-build-" + str(plugin_build)
+        pltfrm = str(key_word).split('_')[0]
+        arch = str(key_word).split('_')[1]
+        plugin_type = str(key_word).split('_')[2]
+        response = s3.list_objects_v2(Bucket=rc_bucket, Prefix=plugin_fullpath)
+        plugin = ""
+        for artifact in sorted(response['Contents'], key=lambda x: x['LastModified'], reverse=True):
+            key = artifact['Key']
+            if pltfrm == "noplatform":
+                pltfrm = ""
+            if arch == "noarch":
+                arch = ""
+            if (plugin_name in key and plugin_ver in key and 
+                    bld_num in key and pltfrm in key and 
+                    arch in key):
+                if key.endswith(plugin_type):
+                    plugin = key
+                    break
+        return plugin
+    except:
+        raise
+
+# Check the Production Path
+def check_prod_location(bucket_name,folder_path):
+    try:
+        plugin_folder = folder_path.split('/')[-2]
+        prefix = folder_path.replace(plugin_folder+"/",'')
+        response = s3.list_objects_v2(Bucket=bucket_name, Delimiter='/', Prefix=prefix)
+        if response.get('CommonPrefixes'):
+            for plg_path in response.get('CommonPrefixes'):
```

Comment:
plg vs plugin, please consider just use plugin here.

Reply:
Changed the naming.